compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1245,7 +1245,7 @@ rustc_queries! {
     /// Returns the attributes on the item at `def_id`.
     ///
     /// Do not use this directly, use `tcx.get_attrs` instead.
-    query item_attrs(def_id: DefId) -> &'tcx [ast::Attribute] {
+    query hir_attrs_for_def(def_id: DefId) -> &'tcx [hir::Attribute] {
```

Comment:
Unnecessary renaming.

Reply:
I discussed these renamings with, I believe it was @BoxyUwU, when I made them. She liked these names better, especially the `for_def` if I remember correctly. I also agree with that. However, whether it should be part of this PR can be debated of course.